config/puma.rb

```diff
@@ -22,3 +22,6 @@
   Sequel::Model.db.disconnect
 end
 
+before_fork do
```

Comment:
Amm this is intresting, I don't know if I would print this log as part of `puma` since the server might fail to start and I'll have this log line hanging... 

Where do we print other configuration properties?

In addition, why we use `before_fork` It will run each time workder duplicate itself, I think it's also a deprecated method

WDYT to combine it with `on_worker_boot` above?
```
Code to run when a worker boots to setup
the process before booting the app.
```
https://github.com/nathansamson/puma/blob/2562379635c01906a3124a7600f593570cac92e0/lib/puma/dsl.rb#L175


Also suggest:
```
Rails.logger.info(LogMessages::Conjur::FipsModeStatus.new(OpenSSL.fips_mode))
```
If `OpenSSL.fips_mode` will always return true/false

Reply:
`on_worker_boot` will run in a worker when it boots 
see: https://www.rubydoc.info/gems/puma/Puma%2FDSL:on_worker_boot

`before_fork `run on master before  forks workers (once on boot) 
see: https://www.rubydoc.info/gems/puma/Puma/DSL#before_fork-instance_method

seems that we don't printing any configuration properties to the logs

Reply:
@micahlee @sashaCher 
Can you share your opinion on using `puma.rb` and `before_fork` to display the logs message, is it the right place?
As brought up we also use `webrick` in some cases

Reply:
> Can you share your opinion on using puma.rb and before_fork to display the logs message, is it the right place?

Hmm.. I don't know about the Rails and Puma/Webrick relation to know the answer to this. @jonahx @jtuttle or @jvanderhoof, do any of you know or have any thoughts?

Reply:
Why do you need to print this when puma forks a process? Can you remove the `before_fork` and just have it print once during initialization or does that not work for some reason?

Reply:
It's works fine and print this log once during initialization without the `before_fork` method, added new commit to solved this issue.
@jtuttle @micahlee @uCatu